codegenerator/cli/src/commands.rs

```diff
@@ -133,21 +133,6 @@ pub mod codegen {
         Ok(exit3)
     }
 
-    pub async fn npx_codegen(
-        envio_version: String,
-        project_paths: &ParsedProjectPaths,
-    ) -> anyhow::Result<()> {
-        let package = format!("envio@{}", envio_version);
-        let args = vec!["--yes", package.as_str(), "codegen"];
-        execute_command("npx", args, &project_paths.project_root)
```

Comment:
Thanks to the new local-bin.mjs setup managed to find this old code, where we ran fuel codegen via npx 😅